crates/telio-traversal/src/session_keeper.rs

```diff
@@ -74,10 +97,14 @@ impl SessionKeeper {
 
         Ok(Self {
             task: Task::start(State {
-                pinger_client_v4: client_v4,
-                pinger_client_v6: client_v6,
-                keepalive_actions: RepeatedActions::default(),
+                pingers: Pingers {
+                    pinger_client_v4: client_v4,
+                    pinger_client_v6: client_v6,
+                },
+                batched_actions: HashMap::new(),
+                nonbatched_actions: RepeatedActions::default(),
```

Comment:
where do we run non batched actions 🤔

Reply:
So that was a bug actually, it was caught by natlab, fix this one. SessionKeeper's `tokio::select!` now awaits for bothh: bathed and nonbatched futures

Reply:
I think this means that we are missing some tests that should have failed.